components/compliance-service/reporting/relaxting/migrations.go

```diff
@@ -359,22 +359,22 @@ func (backend ES2Backend) getScanDateRange(indexPrefix string) (*time.Time, *tim
 	return &earliestScanDate, &mostRecentScanDate, nil
 }
 
-func (backend ES2Backend) reindex(src, dest, reindexScript, srcDocType string) (*elastic.BulkIndexByScrollResponse, bool, error) {
```

Comment:
We found that the `elastic.BulkIndexByScrollResponse` was not being used and we do not have access to it when starting the reindex with `DoAsync`